### PR TITLE
T266675 fix gallery storybook and add error state

### PR DESCRIPTION
**Pull request by hueitan**
Phabricator Ticket : https://phabricator.wikimedia.org/T266675#6623145

(1)
image

(2)
image

### CODE REVIEW

File: src/stories/gallery.stories.js [hunk 1]
```diff
@@ -33,23 +33,42 @@ export const Fullscreen = ( { lang, dir } ) => {
...
+			title: 'File:English_Wikipedia_main_page_20011217.jpg'
+		}
+	]
+
+	showFullscreenGallery( mediaItems, '', lang, dir, container )
+
+	return container
+}
 
+export const Error = ( { lang, dir } ) => {
```
Comment (stephanebisson) [inline src/stories/gallery.stories.js:64]: This renders as just black for me.

Comment (medied) [reply src/stories/gallery.stories.js:64]: Same for me

Comment (hueitan) [reply src/stories/gallery.stories.js:64]: I remember sometimes I also see a blank error page, but I can't reproduce it now.

can you load the error state directly in http://localhost:6006/?path=/story/gallery--error? checkout the fullscreen story first then error.

Comment (hueitan) [reply src/stories/gallery.stories.js:64]: well, please try it again with the latest update

Comment (stephanebisson) [reply src/stories/gallery.stories.js:64]: It works for me now but it's weird. It should work with any number of images.

Comment (hueitan) [reply src/stories/gallery.stories.js:64]: yes, this is fixed here! https://github.com/wikimedia/wikipedia-preview/pull/93

**Pull request merged**